src/models/thelio-mega-r1.0/repairs.md

```diff
@@ -0,0 +1,538 @@
+2. To remove the back CPU shroud fan, unscrew the four screws holding the fan brackets into the shroud (highlighted green below).
+
+![CPU shroud inner screws](./img/cpu-shroud-screws-inner.webp)
+
+3. Free the cable from the velcro loop (highlighted blue above), then unscrew the four screws holding the brackets onto the fan.
+
+![CPU shroud back fan screws](./img/cpu-shroud-fan-screws-back.jpg)
+
+4. To remove the front CPU shroud fan, unscrew the four screws attaching the front of the shroud to the back (highlighted red above).
+5. Then, unscrew the four screws attaching the fan to the shroud.
+
+![CPU shroud front fan screws](./img/cpu-shroud-fan-screws-front.jpg)
+
+### Steps to replace the cooler-mounted CPU fan:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the inner partition](#removing-the-inner-partition), [remove the GPU brace and GPUs](#replacing-a-gpu), [remove the CPU shroud](#removing-the-cpu-shroud), and [remove the top crossbar](#removing-the-top-crossbar).
+2. Pull the corners of the fan's top clip away from the heatsink it's held to.
+
+![CPU cooler fan top clip](./img/cpu-cooler-fan-clip.jpg)
+
+3. Repeat the process for the bottom clip, then slide the fan down and out of the CPU cooler.
+4. Unplug the fan from the Thelio-IO board.
+    - The CPU cooler fan plugs directly into the `CPUOUT0` port on the bottom Thelio-IO board.
+5. When reinstalling the CPU fans, all three fans should be oriented so the side with the non-spinning cover faces the back of the machine, while the spinning side faces the front of the machine.
+
+## Replacing the CPU cooler and CPU:
+
+The CPU cooler dissipates heat from the CPU to the heatsink, where the CPU fans expel it from the system. Depending on your climate and the age of the machine, replacing the thermal paste between the CPU and the cooler/heatsink may help the system run cooler.
+
+The CPU uses an AMD sTRX4 socket.
+
+**Tools required:** Cross-head (Phillips) screwdriver (long), torx screwdriver, thermal paste  
+**Time estimate:** 35 minutes  
+**Difficulty:** High <span style="color:red;">●</span>
+
+### Steps to remove the CPU cooler/thermal paste:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the inner partition](#removing-the-inner-partition), [remove the GPU brace and GPUs](#replacing-a-gpu), [remove the CPU shroud](#removing-the-cpu-shroud), [remove the top crossbar](#removing-the-top-crossbar), and [remove the cooler-mounted CPU fan](#steps-to-replace-the-cooler-mounted-cpu-fan).
+2. While holding the cooler in place so it doesn't fall, reach a cross-head (Phillips) screwdriver through the holes in the cooler plate and loosen the two screws holding the cooler onto the mounting bracket.
```

Comment:
> While holding the cooler in place so it doesn't fall, reach a cross-head (Phillips) screwdriver through the holes in the cooler plate and loosen the two screws holding the cooler onto the mounting bracket.

These CPU coolers are supposed to have threaded thumbscrew caps over these holes. I don't think they require a picture of their own, but it might be good to mention that they have to be removed first.

Reply:
I saw that in the upstream cooler manual, but our lab unit doesn't have them.

Reply:
Confirmed with Carl that the caps will be present on customer units, they are just missing from our lab unit. I'll add a note about them.